client/client.js

```diff
@@ -150,8 +150,7 @@ Client.prototype.serviceList = serviceList;
 function serviceDestroy(nameOrId, callback) {
```

Comment:
should this arg be `serviceNameOrId`, or is it different from the other args with that name?